keras_cv/models/backbones/resnet_v2/resnet_v2_backbone_presets.py

```diff
@@ -115,6 +120,7 @@
                 "Trained on ILSVRC 2012 (Imagenet) classification task."
             ),
         },
+        "class_name": "keras_cv.models>ResNetV2Backbone",
```

Comment:
what is the advantage of using `>` over the `.` ?

Reply:
This is just the Keras serialization, i.e., `keras.utils.get_registered_name(ResNetV2Backbone)`. If present, you can get an instance without knowing the backbone class; i.e., `keras.utils.deserialize_keras_object(preset)`